emails/notion-magic-link.tsx

```diff
@@ -0,0 +1,124 @@
+import { Body, Container, Head, Heading, Html, Img, Link, Preview, Text } from '@react-email/components';
+import * as React from 'react';
+
+interface NotionMagicLinkEmailProps {
+    loginCode?: string;
+}
+
+const baseUrl = process.env.VERCEL_URL ? `https://${process.env.VERCEL_URL}` : '';
```

Comment:
Why here `VERCEL_URL` var name?

Reply:
This is the default example of email template.